modules/iopcore/cdvdman/imports.lst

```diff
@@ -41,6 +42,7 @@ I_strrchr
 I_strstr
 I_strncpy
 I_sprintf
+I_memcpy
```

Comment:
`memcpy` should not be imported. Instead smsutils has `mips_memcpy`.
When including smsutils.h, this will automatically define `memcpy` as `mips_memcpy`. I know it's a little strange, but that's how it is, and we should nog add the extra `memcpy`.

Reply:
This is interesting. But we need to be careful that adding this header doesn't break cross-system support. Remember that this code is compiled for EE and IOP, and I already had issues with headers that would compile with one but not the other.
As an alternative, we can change memcpy to ziso_memcpy and have it be implemented externally like ziso_alloc.